bluepyemodel/tools/search_pdfs.py

```diff
@@ -277,19 +277,18 @@ def figure_emodel_parameters_evolution(emodel_metadata, seed, use_allen_notation
     fname = f"{metadata_str}__evo_parameter_density.pdf"
 
     pathname = Path("./figures") / emodel_metadata.emodel / "parameter_evolution" / fname
-    pathname_val = Path("./figures") / emodel_metadata.emodel / "parameter_evolution" / fname
 
-    return pathname, pathname_val
+    return pathname
 
 
 def search_figure_emodel_parameters_evolution(emodel_metadata, seed, use_allen_notation=True):
     """Search for the pdf representing the evolution of the parameters of an emodel"""
 
-    pathname, pathname_val = figure_emodel_parameters_evolution(
+    pathname = figure_emodel_parameters_evolution(
         emodel_metadata, seed, use_allen_notation=use_allen_notation
     )
 
-    return [search_figure_path(str(pathname)), search_figure_path(str(pathname_val))]
+    return search_figure_path(str(pathname))
```

Comment:
I think you should keep it inside the list or change the logic here: https://github.com/BlueBrain/BluePyEModel/blob/022b973b9e5901caa16234e0ab197585a2510249/bluepyemodel/emodel_pipeline/emodel.py#L142
otherwise it will split the path into individual characters

Reply:
solved in recent commit